### ISSUE TITLE
Eleven viz sliders drag a parameter the primitive never reads

### ISSUE THREAD

**Issue by opendroid** [opendroid/the-infinity#177]
Found while verifying the Evaluation batch (#61).

## The defect

Eleven published concept pages render a slider that does nothing. The reader drags it, the number in the caption row changes, and the figure does not move — while the caption explicitly promises that it will ("Drag the vocabulary size to watch the number move", "Drag the length to find where the two curves stop agreeing").

Three of the five primitives filter the node's `viz.params` through a typed reader before using them:

- `losscurve.ts` → `paramsFrom()` keeps `steps, lr, batch, warmup, decay, holdout, params`
- `attention.ts` → `shapeFrom()` keeps `tokens, keys, lookahead, heads, decay, sink`
- `RouterDispatch.tsx` → `read()` keeps `experts, top_k, capacity_factor`

Anything else is silently dropped. `budget-split` and `update-spectrum` take the control's value positionally, so they cannot have this bug.

## Why nothing caught it

Both existing checks are spelling checks. `nodes.test.ts` asserts:

```
it('names a viz parameter that actually exists', () => {
  expect(Object.keys(node.viz.params)).toContain(control.name);
});
```

and `validate-content.mjs` asserts the same thing. The parameter *did* exist. Nothing read it. This is the "check that passes for the wrong reason" pattern again — the assertion's name claims more than its body tests.

## Affected

Two distinct causes, both producing an inert slider.

**A control the reader ignores entirely** — the name is not in the primitive's allowlist:

| node | control | tier |
|---|---|---|
| `perplexity` | `vocab` | frontier |
| `emergent-abilities` | `tokens` | frontier |
| `chain-of-thought` | `chain` | frontier |
| `rlhf` | `beta` | **verified** |
| `direct-preference-optimization` | `beta` | **verified** |
| `diffusion-model` | `timesteps` | frontier |
| `vision-transformer` | `data` | frontier |

**A control the reader accepts but the drawing cannot show** — `steps` is read, but the curve is sampled over a normalised `u ∈ [0,1]`, so run length is invisible by construction unless `warmup > 0`:

| node | control | tier |
|---|---|---|
| `learning-rate-schedule` | `steps` | **verified** |
| `loss-function` | `steps` | **verified** |
| `overfitting` | `steps` | **verified** |
| `pretraining` | `steps` | **verified** |

Six of the eleven are `verified`, which is the part worth sitting with: a human read those pages and the inert slider was not visible from the JSON.

`vision-transformer` is worse than inert — its caption describes two curves ("a transformer and a convolutional model") and `lossCurve` returns `compare: null` for its params, so only one is drawn.

## Two more from the same read

Not inert, but the node disagrees with the primitive's stated contract:

- **`calibration`** drags `buckets` through `update-spectrum`, whose contract is *"raising the control flattens the right-hand group."* More buckets does not flatten a confidence distribution; the figure asserts something untrue.
- **`in-context-learning`** splits `shots: 8` against `share_rest: 256` in `budget-split`, whose contract is *"`share_rest` names the fixed remainder **in the same units**."* Eight shots against 256 tokens are not the same unit.

## Acceptance criteria

- [ ] A test that runs each primitive's own arithmetic at both ends of every node's control range and asserts the drawn output differs — behavioural, so it cannot drift from the readers the way a name list would
- [ ] That test excludes the params echoed back for the caption row, or a slider passes by changing only the printed number
- [ ] All eleven nodes drag a parameter the primitive reads, with captions rewritten to promise only what renders
- [ ] `calibration` and `in-context-learning` agree with their primitive's contract
